as-vending/functions/output.go

```diff
@@ -338,18 +354,18 @@ func (vendingState *VendingState) getCardAuthInfo(edgexcontext *appcontext.Conte
 	var auth OutputData
 	_, err = utilities.ParseJSONHTTPResponseContent(resp.Body, &auth)
 	if err != nil {
-		edgexcontext.LoggingClient.Error("Could not read response body from AuthenticationEndpoint")
+		lc.Error("Could not read response body from AuthenticationEndpoint")
```

Comment:
it is probably beneficial for trouble shooting purpose to include error in the logging message:
```suggestion
		lc.Errorf("Could not read response body from AuthenticationEndpoint: %v", err)
```